SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -420,3 +463,43 @@ FString USpatialGDKEditorSettings::GetCookAndGenerateSchemaTargetPlatform() cons
 	// Return current Editor's Build variant as default.
 	return FPlatformProcess::GetBinariesSubdirectory();
 }
+
+FString FSpatialLaunchConfigDescription::GetTemplate(bool bUseDefault) const
+{
+	if (bUseDefault)
+	{
+#if PLATFORM_MAC
+		if (GetDefault<USpatialGDKEditorSettings>()->GetPrimaryRegionCode() == ERegionCode::CN)
+		{
+			return SpatialGDKServicesConstants::PinnedChinaCompatibilityModeRuntimeTemplate;
+		}
+		else
+		{
+			return SpatialGDKServicesConstants::PinnedCompatibilityModeRuntimeTemplate;
+		}
+#endif
+		switch (GetDefault<USpatialGDKEditorSettings>()->GetSpatialOSRuntimeVariant())
```

Comment:
```suggestion
#if PLATFORM_MAC
               switch(ESpatialOSRuntimeVariant::CompatibilityMode)
#else
		switch (GetDefault<USpatialGDKEditorSettings>()->GetSpatialOSRuntimeVariant())
#endif
```

Reply:
Nice idea, didn't think that would work with the `default`. Done!